cg/models/orders/excel_sample.py

```diff
@@ -1,5 +1,7 @@
+import datetime
 from typing import List, Optional
 
+import dateutil.parser
```

Comment:
`dateutil` has been removed from the dependencies so please don't add it again!

Reply:
good catch!

Reply:
Deadpendency should have the option to fail the build if there's suddenly more code relying on dead dependencies, now it's just failing all the time 🤕

Reply:
we still have a lot of dependencies on `dateutil` still or is my branch just containing old code? 
<img width="1003" alt="image" src="https://user-images.githubusercontent.com/6697636/117947233-4bf30b00-b310-11eb-9afd-280b8370dd8c.png">

Reply:
:( I spent a lot of time removing that before so someone must have added it again...

Reply:
I would recommend to not use `dateutils` so that it will be easier to remove it again

Reply:
I removed it